provisioning/transport/http/src/ProvisioningTransportHandlerHttp.cs

```diff
@@ -148,10 +148,43 @@ public ProvisioningTransportHandlerHttp()
 
                     cancellationToken.ThrowIfCancellationRequested();
 
-                    operation = await client.RuntimeRegistration.OperationStatusLookupAsync(
+                    try
+                    {
+                        operation = await client.RuntimeRegistration.OperationStatusLookupAsync(
                         registrationId,
                         operationId,
                         message.IdScope).ConfigureAwait(false);
+                    }
+                    catch(HttpOperationException oe)
+                    {
+                        bool isTransient = oe.Response.StatusCode >= HttpStatusCode.InternalServerError
+                            || (int)oe.Response.StatusCode == 429;
```

Comment:
use the HttpStatusCode enum here so you don't have to cast it to Int

Reply:
agreed, I'll take care of that in any follow up PR